### DISCUSSION TITLE
Implement Classic Navigation+Address Bar with ExplorerFrame

### DISCUSSION THREAD

**Discussion by TheDolphinSlayer** [valinet/ExplorerPatcher#2161] [Ideas]
When using ExplorerFrame.dll from Windows 11 Beta Builds, its possible to have the windows 10 explorer again.
This works on Windows 11 from RTM to 23H2, to fix Quick Access you need to use ViveTool and disable the tabs!

If Valinet could just use the addressbar and navigation buttons from ExplorerFrame.dll that'd be okay as this does disable the windows 11 command bar! But it functions properly, restores the seperator between the addressbar and refresh button, and has PROPER compact, not the bullshit windows 11 comes with.

THIS WAS NOT DISCOVERED BY ME, discovered by ITTR on WinClassic Fourms! If needed I can send said ExplorerFrame.dll

https://archive.org/download/21337.1010.210312-1700.-rs-prerelease-flt-clientmulti-x-64-fre-en-us/21332.1010.210307-1057.RS_PRERELEASE_FLT_CLIENTMULTI_X64FRE_EN-US.ISO

If you want said beta build with ExplorerFrame.dll

![image](https://github.com/valinet/ExplorerPatcher/assets/73983610/1184056d-56c8-4238-a661-12cb49d427bf)

**Comment by pyrates999** [comment]
Insider builds are not supported.

**Reply by TheDolphinSlayer** [reply]
What're you going on about

**Reply by pyrates999** [reply]
https://www.elevenforum.com/t/windows-11-insider-dev-build-23545-1000-22h2-sept-13.17927/

**Comment by Amrsatrio** [comment]
Is this what you mean?

![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/3413185c-bf02-4d07-a67f-19a6c7ae9518)

**Comment by TheDolphinSlayer** [comment]
No. I am 100% aware that the windows 10 ribbon is still in windows 11, that's the whole point of explorerpatcher. But, when Microsoft added the option to make explorer view compact, they left out the navigation back and forward buttons. The explorerframe.dll that I'm talking about, is from an early version of windows 11 before they changed the UI scaling to how they are now. The dll I sent in that iso build will literally bring back the full compact look from windows 10, and not the half assed one in windows 11. AKA, compact folder spacing, tall address bar, and wide ass back and forward buttons. I'd like for Valinet to implement this, instead of the hassle of going back and forth with explorerframes.
